scripts/install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Any subsequent(*) commands which fail will cause the shell script to exit immediately
+set -e
+
+# Supported versions of node: v4.x, v5.x
+NODE_VERSION="v4.*\|v5.*"
+
+# Ensure we are using sudo
+if [ "$(whoami)" != "root" ];
+then
+	echo "This script requires root permissions, try: sudo ./${0##*/} "
+	exit 0
+fi
+
+cat << "EOF"
+ ________  _____ ______   ________  ________  _________         
+|\   ____\|\   _ \  _   \|\   __  \|\   __  \|\___   ___\       
+\ \  \___|\ \  \\\__\ \  \ \  \|\  \ \  \|\  \|___ \  \_|       
+ \ \_____  \ \  \\|__| \  \ \   __  \ \   _  _\   \ \  \        
+  \|____|\  \ \  \    \ \  \ \  \ \  \ \  \\  \|   \ \  \       
+    ____\_\  \ \__\    \ \__\ \__\ \__\ \__\\ _\    \ \__\      
+   |\_________\|__|     \|__|\|__|\|__|\|__|\|__|    \|__|      
+   \|_________|                                                 
+                                                                
+ _____ ______   ___  ________  ________  ________  ________     
+|\   _ \  _   \|\  \|\   __  \|\   __  \|\   __  \|\   __  \    
+\ \  \\\__\ \  \ \  \ \  \|\  \ \  \|\  \ \  \|\  \ \  \|\  \   
+ \ \  \\|__| \  \ \  \ \   _  _\ \   _  _\ \  \\\  \ \   _  _\  
+  \ \  \    \ \  \ \  \ \  \\  \\ \  \\  \\ \  \\\  \ \  \\  \  
+   \ \__\    \ \__\ \__\ \__\\ _\\ \__\\ _\\ \_______\ \__\\ _\ 
+    \|__|     \|__|\|__|\|__|\|__|\|__|\|__|\|_______|\|__|\|__|
+
+EOF
```

Comment:
It's a site l that I've been using for ~8 years:
http://patorjk.com/software/taag

It never goes out of style!